packages/esm-patient-test-results-app/src/filter/filter-types.ts

```diff
@@ -22,12 +31,29 @@ export interface FilterLeafProps {
   leaf: Observation;
 }
 
+export interface TreeParents {
+  [key: string]: Array<string>;
+}
+
+export interface TreeCheckboxes {
+  [key: string]: boolean;
+}
+
+export interface TreeTests {
+  [key: string]: TreeNode;
+}
+
+export interface TreeNode {
```

Comment:
One of these TreeNode definitions ought to be deleted.

Reply:
Done